src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1110,11 +1110,40 @@ bool entitiesQuery
     }
 
     /* Process query result */
-    while (cursor->more()) {
+    while (cursor->more())
+    {
+        BSONObj                 r    = cursor->next();
+        std::string             err  = r.getStringField("$err");
+        ContextElementResponse* cer  = new ContextElementResponse();
+
+        if (err != "")
+        {
+          //
+          // We can't return the error 'as is', as it may contain forbidden characters.
```

Comment:
I tried to return the value 'as is' and it failed. That's why I invented this mechanism to return something similar, at least.

NTC?